definitions/EiffelIssueDefinedEvent/1.0.0.yml

```diff
@@ -0,0 +1,114 @@
+$schema: http://json-schema.org/draft-04/schema#
+_abbrev: ID
+_description: The EiffelIssueDefinedEvent declares that an issue has
+  been created in someexternal issue management software. ID is semantically
```

Comment:
```suggestion
  been created in some external issue management software. ID is semantically
```

Reply:
Fixed in 888fdaa893e88ed28b6374edf2a6aa24539ccfec.